test/unit.js

```diff
@@ -0,0 +1,273 @@
+                tagName = helperElement('x-terminal-hidden').safe;
+                var webComponentConstructor = skate(tagName, {
+                    render: function (elem) {
+                        elem.innerHTML = '<div></div>';
+                        if (elem.hidden) {
+                            elem.firstChild.setAttribute('aria-hidden', '');
+                        } else {
+                            elem.firstChild.removeAttribute('aria-hidden');
+                        }
+                    },
+                    properties: {
+                        hidden: {
+                            set: function (elem) {
+                                skate.render(elem);
+                            }
+                        }
+                    }
+                });
+
+                ReactComponent = makeReactComponent(webComponentConstructor)
+            });
+
+            it('the web component renders props through the property', function () {
+                renderApp(
+                    React.createClass({
+                        render: function () {
+                            return <ReactComponent hidden="true"/>;
+                        }
+                    })
+                );
+
+                expect(getWebComponent().firstChild.hasAttribute('aria-hidden')).to.be.true;
+            });
+        });
+    });
+
+    describe('(React > React wrapped container web component', function () {
+
+        function isReactRoot (elem) {
+            return elem.hasAttribute('data-reactid') && elem.getAttribute('data-reactid').split('.').length === 2;
+        }
+
+        beforeEach(function () {
+            tagName = helperElement('x-container').safe;
+            var webComponentConstructor = skate(tagName, {
+                render: function (elem) {
+                    elem.innerHTML = '<div class="parent"><div class="content"></div></div>';
+                },
+                properties: {
+                    content: {
+                        get: function content (elem) {
+                            return elem.querySelector('.content').firstChild;
+                        },
+
+                        set: function content (elem, changeData) {
+                            const container = elem.querySelector('.content');
+                            const reactTree = changeData.newValue;
+                            container.innerHTML = '';
+                            if (reactTree) {
+                                container.appendChild(reactTree);
+                            }
+                        }
+                    }
+                }
+            });
+
+            ReactComponent = makeReactComponent(webComponentConstructor);
+        });
+
+        it('> DOM element): content is redirected to the correct container', function () {
+            renderApp(
+                React.createClass({
+                    render: function () {
+                        return <ReactComponent>
```

Comment:
You can wrap your jsx expression in parenthesis if you want cleaner indentation:

```js
return (
  <div>
    content...
  </div>
);
```